package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Gnosis Safe default apps list",
   "main": "public/app-list.json",
   "scripts": {
-    "build": "rimraf public && mkdir -p public && cross-env node src/write.js > public/gnosis-default.applist.json",
```

Comment:
Why do you remove cross-env? This usually ensures that ENV parameters are used correctly independently of the OS

Reply:
I ran into a weird bug and was trying to find the reason. Forgot to bring it back, thanks